modules/drivers/video/conf/video_right_rear.pb.txt

```diff
@@ -0,0 +1,30 @@
+
+udp_port: 2007
+frame_id: "video_right_rear"
+pixel_format: "yuyv"
+trigger_param: "f2ff"
+metric_error_code: 31
+record:0
+width: 1920
+height: 1080
+frame_rate: 5
```

Comment:
why fisheyes frame rate lower than others?

Reply:
it is come from mainline, perception team decide it, you can change is as apollo requirement